analysis/pseudobulk-bulk-prediction/exploratory-notebooks/view-ora-results.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "Visualize ORA results"
+author: Stephanie J. Spielman
+date: "`r Sys.Date()`"
+output:
+  html_notebook:
+    toc: true
+    toc_depth: 3
+    code_folding: hide
+---
+
+This notebook visualizes the odds ratios from over-representation analysis (ORA) performed in `../notebooks/perform-ora.Rmd`, where adjusted p-values were calculated using BH-correction following a permutation test with 10,000 replicates.
+
+Each odds ratio can be interpreted interchangeably as follows, where "outlier" indicates that the gene is in the model residual outliers in either the positive (bulk) or negative (pseudobulk) direction.
+A bulk outlier is defined as $> 2.5\sigma_{summarized\_residual}$ , and a pseudobulk outlier is defined as $< -2.5\sigma_{summarized\_residual}$ , where the `summarized_residual` values are the median residuals for each gene.  
+
+* The odds an outlier is a cell-type marker gene compared to non-outlier
+* The odds an cell type marker gene is an outlier compared to other genes
+
+## Setup
+
+```{r setup}
+renv::load()
```

Comment:
This really shouldn't be needed, unless you are rendering this in a particularly odd way. RStudio will automatically load the renv for the project.

Reply:
So, I have found myself accidentally "un-previewing" output without realizing it so many times over the years that I now default to rendering notebooks on the command line, and `renv::load()` is needed for that. It's definitely not needed when running within RStudio though